recipes/CapabilityPatternNode/README.md

```diff
@@ -0,0 +1,15 @@
+# CapabilityPatternNode
+
+A skeletal version of an application structuring pattern
+
+See the readme in https://github.com/afcondon/purescript-capability-pattern
```

Comment:
I'm wondering whether we should inline that repo's readme content here. @milesfrain your thoughts?

Reply:
Yes. That would probably be easier to maintain

Reply:
i'll do that now, so

Reply:
I'll edit it to make it less "personal repo chatty" and more "cookbook semi-formal"